docs/build/advanced-concepts/flix.md

```diff
@@ -8,18 +8,49 @@ Flow Interaction Templates is a standard for how contract developers, wallets, u
 
 Interaction Templates provide a way to use and reuse existing scripts and transactions, as well as to provide more metadata such as a human-readable title and description of what the transaction or script will do, which can be used by the developer as well as the user of the application. 
 
-By using FLIX transactions and scripts, develoeprs don’t have to write their own for common operations!
+By using FLIX transactions and scripts, developers don’t have to write their own for common operations!
 
 Read more about the design and purpose of FLIX in the [FLIP](https://github.com/onflow/flips/blob/main/application/20220503-interaction-templates.md)
 
 ## Using FLIX
 
-Flow makes FLIX available through an API available at flix.flow.com. Other community run APIs are available, such as flix.ecdao.org. 
+Flow makes FLIX available through an API available at flix.flow.com. 
 
 You can query a FLIX API to get an Interaction Template. An example query looks like: https://flix.flow.com/v1/templates?name=transfer-flow
 
 You can read more about how to query a FLIX API in the documentation available here: [https://github.com/onflow/flow-interaction-template-service](https://github.com/onflow/flow-interaction-template-service)
 
+::: info
+
+The FLIX working group is currently working on a protocol to publish FLIX templates on-chain.
+
+:::
+
+### Example
+
+How to integrate FLIX across different developer teams? For this example there are two github repositories. 
+ - (smart contracts) [https://github.com/onflow/hello-world-flix](https://github.com/onflow/hello-world-flix)
+ - (web development) [https://github.com/onflow/hello-world-web](https://github.com/onflow/hello-world-web)
+
+The Smart contract developer creates FLIX templates and makes them available in github, these can be versioned. Example is `v0.1.0` release, the templates are available for a specific version. In this example the templates are located at:
+ - https://github.com/onflow/hello-world-flix/blob/v0.1.0/cadence/templates/ReadHelloWorld.template.json
+ - https://github.com/onflow/hello-world-flix/blob/v0.1.0/cadence/templates/UpdateHelloWorld.template.json
+
+Developers can use FLIX templates from the smart contract github to interact with their smart contracts. They simply need the FLIX template URLs to create binding files (TypeScript or JavaScript). One major benefit is the web developers don't need to learn Cadence or copy Cadence to their repository in order to integrate with existing smart contracts. 
+
+TypeScript code generated from templates:
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/readHelloWorld.ts
+- https://github.com/onflow/hello-world-web/blob/main/app/cadence/updateHelloWorld.ts
+
+::: warning
+
+manually added @ts-ignore, fixes to fcl types will correct these linting issue
```

Comment:
could you rephrase this sentence? I'm not sure what it means

Reply:
updated PR, thanks